ci-build.py

```diff
@@ -77,6 +77,7 @@
         'PWD': build_dir,
         })
     sp.check_call(f'tox -e py37 -- -n auto -v -x', shell=True, cwd=build_dir, env=tox_env)
+    sp.check_call(f'tox -e flake8', shell=True, cwd=build_dir, env=tox_env)
```

Comment:
I'm not sure if flake8 should really be part of `--run-tests`... that doesn't make a lot of sense.

We should either rename the CLI argument (but I wouldn't know a good name) or introduce a `--flake8` argument.
I'd be fine with the latter, but this won't scale well *if* we are going to add many more things like that.

Alternatively, we could offer something like `--checks=iwyu,tests,flake8`. Unfortunately, argparse doesn't seem to offer that out of the box.

Reply:
I added an extra flag for flake8 and extracted the whole tox environment preparation and execution